products/sle/main.pm

```diff
@@ -802,6 +802,12 @@ elsif (get_var("VIRT_AUTOTEST")) {
         loadtest "virt_autotest/update_package";
         loadtest "virt_autotest/reboot_and_wait_up_normal";
     }
+    if (check_var('ARCH', 's390x') and (get_var("VIRT_PRJ1_GUEST_INSTALL"))) {
+        loadtest "virt_autotest/login_console";
+        loadtest "virt_autotest/install_package";
+        loadtest "virt_autotest/update_package";
+        loadtest "virt_autotest/reboot_and_wait_up_normal";
```

Comment:
These four modules are already covered below in lines https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/6549/files#diff-e1899612c0794cc2244eca9680590031R818 until https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/6549/files#diff-e1899612c0794cc2244eca9680590031R823 so repeating it here for s390x looks confusing and not very clean.

Reply:
yes, had made code changed as your mentioned. now, do verification run, and after that, will update this PF ASAP.

Reply:
Made code changed as your mentioned, updated this PR and 
Verification Run:
http://10.67.17.251/tests/90